nitrite.go

```diff
@@ -264,7 +308,7 @@ func Verify(data []byte, options VerifyOptions) (*Result, error) {
 		return nil, ErrMandatoryFieldsMissing
 	}
 
-	if !options.AllowSelfSignedCert && nil == doc.CABundle {
```

Comment:
Removing the support for self-signed certs. We had those in `nitriding` but don't support self-signed nitro attestations in newman

Reply:
Based on conversation with @dlm, will bring this functionality back and test on a self-signed attestation

Reply:
done